src/psyclone/tests/test_files/dynamo0p3/weighted_proj_theta2_kernel_mod.F90

```diff
@@ -1,11 +1,43 @@
+! Modifications copyright (c) 2017, Science and Technology Facilities Council
```

Comment:
Please delete this line as it is written further down the code.

Reply:
Deleted.